src/cromshell/status/command.py

```diff
@@ -114,34 +115,74 @@ def main(config, workflow_id):
     return ret_val
 
 
-def get_metadata_status_summary(workflow_metadata):
+def get_all_call_values(nested_dictionary):
+    """Summarizes a given workflow metadata dictionary into a
+    new dictionary with only task name and status"""
+    temp_dic1 = {}
+    temp_dic2 = {}
+    temp_dic3 = {}
+    for key, value in nested_dictionary.items():
+        # Skip any keys that have values that are strings, because we are note
+        # interested in them
+        if type(value) is str:
+            continue
+        # If a dictionary value is encountered then we iterate through the dictionary
+        # to find call statuses
+        if type(value) is dict:
+            temp_dic1 = get_all_call_values(value)
+        # If a list value is encountered then those are the calls and their status,
+        # we'll want to add the key and value to the temp dictionary
+        if type(value) is list:
+            # Each item in the list is the shard of a call so we'll itterate through
+            # the list and collect info on all shards
+            temp_dic3[key] = []
+            for i, shard in enumerate(value):
+                calls_status = shard
+                # If a key in the shard dictionary has the term "subWorkflowMetadata"
+                # then this is a subworkflow so we iterate through the dictionary to
+                # find call statuses
+                if "subWorkflowMetadata" in calls_status.keys():
+                    temp_dic2 = get_all_call_values(calls_status["subWorkflowMetadata"])
+                    # Remove "Scatter" key from dict
+                    if key in temp_dic3.keys():
+                        temp_dic3.pop(key)
+                # If there isn't a sub workflow then we add the call name combined
+                # with the shard index as a key and its metadata dictionary as its
+                # value
+                if "subWorkflowMetadata" not in calls_status.keys():
```

Comment:
could be just the `else` block of line 144

Reply:
done